ingestion/setup.py

```diff
@@ -34,7 +34,7 @@ def get_long_description():
     "geoalchemy2": "GeoAlchemy2~=0.12",
     "google-cloud-storage": "google-cloud-storage==1.43.0",
     "great-expectations": "great-expectations~=0.15.0",
-    "grpc-tools": "grpcio-tools==1.47.2",
+    "grpc-tools": "grpcio-tools==1.51.1",
```

Comment:
this is causing couple of conflicts, can we add flexibility here?
>=1.47.2
something like that?

Reply:
@ayush-shah  Fixed